.github/workflows/publish-release-assets.yml

```
@@ -0,0 +1,214 @@
+          - target: aarch64-apple-darwin
+            asset_name: meilisearch-macos-apple-silicon
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Installing Rust toolchain
+        uses: dtolnay/rust-toolchain@1.85
+        with:
+          profile: minimal
+          target: ${{ matrix.target }}
+      - name: Cargo build
+        uses: actions-rs/cargo@v1
```

actionlint (1.7.7)

the runner of "actions-rs/cargo@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)